lib/inspec-vault/input.rb

```diff
@@ -9,19 +9,25 @@ class Input < Inspec.plugin(2, :input)
     attr_reader :mount_point
     attr_reader :path_prefix
     attr_reader :vault
+    attr_reader :priority
 
     def initialize
       @plugin_conf = Inspec::Config.cached.fetch_plugin_config("inspec-vault")
 
       @mount_point = fetch_plugin_setting("mount_point", "secret")
       @path_prefix = fetch_plugin_setting("path_prefix", "inspec")
+      @priority = fetch_plugin_setting("priority", 60).to_i
```

Comment:
Why the `to_i`?

I also don't see a test that isn't using this default, unless I'm not understanding it.

Reply:
to_i is due to this value sometimes coming from an environment variable (in which case it is a string). Internally, it must be numeric.

You're right, there is no test on having a default of 60; I'll have to think about how to test that.

Reply:
K, I've added a set of threshold tests around the default value.